Example/Rudder-Firebase/RudderConfig/SampleRudderConfig.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>WRITE_KEY</key>
+	<string></string>
+	<key>PROD_DATA_PLANE_URL</key>
+	<string></string>
```

Comment:
Let's add the steps regarding how to use this file and also add some dummy values to make the example more meaningful